### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by AndriyPT82**
https://andriypt82.github.io/js_2048_game/

### CODE REVIEW

File: src/scripts/main.js [hunk 1]
```diff
@@ -1,3 +1,240 @@
...
+  };
+
+  return true;
+};
+
+function checkForValue2048(table) {
+  return [...table].some(row => {
+    return [...row.children].some(cell => {
+      return cell.innerText === '2048';
+    });
```
Comment (vkryvytskyy) [inline src/scripts/main.js:238]: you don't need inner some, use includes

Comment (vkryvytskyy) [reply src/scripts/main.js:238]: row.includes(2048)

**Review by vkryvytskyy — APPROVED** [review]
GJ

File: src/scripts/main.js (hunk 1 above)
Comment (AndriyPT82) [reply src/scripts/main.js]: Done!

**Pull request closed without merging**